### PR TITLE
Fix Docker build

### PR DESCRIPTION
**Pull request by mjethanandani**

### CODE REVIEW

File: src/validate-and-gen-trees.sh
Comment (jhaas-pfrc) [inline src/validate-and-gen-trees.sh:8]: This will collide with my issue.  Did you want me to hold off committing that?

File: src/yang/ietf-bgp.yang
Comment (jhaas-pfrc) [inline src/yang/ietf-bgp.yang:854]: Don't include this in the fix build commit.

File: src/validate-and-gen-trees.sh
Comment (mjethanandani) [reply src/validate-and-gen-trees.sh:8]: I will resolve this before merging.

File: src/yang/ietf-bgp.yang
Comment (mjethanandani) [reply src/yang/ietf-bgp.yang:854]: I believe this came to me as part of merge I did with v13.

Comment (mjethanandani) [reply src/yang/ietf-bgp.yang:854]: On re-checking, apparently not. Will remove it.

**Review by jhaas-pfrc — APPROVED** [review]

**Pull request merged**
